t/class/inherit.t

```diff
@@ -105,4 +105,12 @@ use A::B;
 EOS
 }
 
+{
+    # https://github.com/Perl/perl5/issues/20891
+    class Test6A 1.23 {}
+    class Test6B 1.23 :isa(Test6A) {}
+
+    pass('class declaration permits VERSION and :ATTRS...');
```

Comment:
Maybe test `ok(Test6B->new isa Test6A)` and `is(Test6B->VERSION, 1.23)` as well, to make sure the everything actually works, not just parses?

Reply:
Ah yes. Test added.